__tests__/app.test.js

```diff
@@ -61,13 +62,68 @@ describe("GET /api/articles/:article_id", () => {
   });
   test('GET:404 sends an appropriate status and error meaasge when given a valid bot non-existent id', () => {
     return request(app)
-    .get('/api/articles/123')
-    .expect(404)
-    .then((response) => {
-      expect(response.body.msg).toBe('article does not exist');
-    })
+      .get('/api/articles/123')
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe('article does not exist');
+      })
   })
 });
+
+describe("GET /api/articles", () => {
+  test("200: Responds with an object of all the articles", () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then(({ body }) => {
+      expect(body.articles).toHaveLength(13);
```

Comment:
This is a really important expect statement to make sure we get to the expect statements in our `.forEach`, good to see 🥇